src/translations/en_NO.json

```diff
@@ -1,8 +1,8 @@
 {
-  "ONBOARDING_CONNECT_DD_HEADLINE": "Connect payment",
-  "ONBOARDING_CONNECT_DD_CTA": "Connect payment",
-  "ONBOARDING_CONNECT_DD_PRE_HEADLINE": "Psst, don't forget...",
-  "ONBOARDING_CONNECT_DD_BODY": "Money will be drawn the 27th each month. We only start charging you once your insurance is activated.",
+  "ONBOARDING_CONNECT_DD_HEADLINE": "only two things left to do",
+  "ONBOARDING_CONNECT_DD_CTA": "Connect payment - activate Hedvig",
+  "ONBOARDING_CONNECT_DD_PRE_HEADLINE": "Welcome to Hedvig,",
+  "ONBOARDING_CONNECT_DD_BODY": "We are so happy to have you as a member. To start using your new home insurance, you need to:",
```

Comment:
Shouldn't say `home insurance`  since it might be travel insurance you've sign up for. So just `insurance`/`forsikring` is better. So it should probably be the same for all English translations. And since these should be _translations_  and nothing else I guess we should just say `försäkring`/`insurance` in Sweden as well?